package.json

```diff
@@ -23,6 +25,7 @@
     "@internetarchive/field-parsers": "^0.1.3",
     "@internetarchive/histogram-date-range": "^0.1.6",
     "@internetarchive/infinite-scroller": "^0.1.2",
+    "@internetarchive/local-cache": "^0.2.1",
```

Comment:
blocker: I see this was moved out of devDependencies. is this used in `app-root.ts` ? if so, let's move this back to devDependencies

Reply:
updated